node/narwhal/committee/src/prop_tests.rs

```diff
@@ -108,7 +108,8 @@ impl Arbitrary for ValidatorSet {
     type Strategy = BoxedStrategy<ValidatorSet>;
 
     fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
-        validator_set(any_valid_validator(), size_range(4..=MAX_COMMITTEE_SIZE as usize)).boxed()
+        // use minimal validator set to speed up tests that require signing from the committee members
+        validator_set(any_valid_validator(), size_range(4..=4usize)).boxed()
```

Comment:
maybe we should at least test 1 more number? or is the current setup hardcoded to a 4-member committee only?

Reply:
This is the `Arbitrary` implementation for `ValidatorSet`, so it generates the "standard" generation strategy for use in external proptests. There are additional (internal) strategies for testing different `ValidatorSet` setups.